opentracing-noop/src/main/java/io/opentracing/noop/NoopTracer.java

```diff
@@ -26,6 +25,11 @@
     final static NoopTracer INSTANCE = new NoopTracerImpl();
 
     @Override
+    public ScopeManager scopeManager() {
+        return null;
```

Comment:
This should return `NoopSpanManager.INSTANCE`, otherwise will cause apps using active spans a problem.

Reply:
Makes sense. Will update it for the next RC.